Formula/nonpareil.rb

```diff
@@ -0,0 +1,34 @@
+class Nonpareil < Formula
+  desc "Estimates covverage in metagenomic datasets"
+  homepage "http://enve-omics.ce.gatech.edu/nonpareil"
+
+  # doi "10.1093/bioinformatics/btt584"
+  url "https://github.com/lmrodriguezr/nonpareil/archive/v3.3.1.tar.gz"
+  sha256 "6fc9caf915f80704876fcd7b0a1fb3beda664fdb840fa0866bdfa1cb76df1a34"
+  head "https://github.com/lmrodriguezr/nonpareil.git"
+
+  depends_on "r"
+  depends_on "open-mpi" => :optional
+
+  def install
+    r_library = lib/"R"/r_major_minor
+    r_library.mkpath
```

Comment:
Could the R package be installed instead in `libexec`?
Would `nonpareil` be able to find the R package installed there?

Reply:
Hmmm... I'm not sure if that would be an option (even though I'd prefer if it goes into `libexec`). Nonpareil is executed in two stages: first the `nonpareil` binary is executed to produce redundancy files, and then those files are analyzed by the user with the help of the `Nonpareil` R package (from within R) to produce plots and compute statistics.

Reply:
You can create a shell script that sets the environment variable `R_LIBS_USER` and then runs R using `write_env_script`. See how this is done for `circos` and Perl for example
https://github.com/brewsci/homebrew-bio/blob/master/Formula/circos.rb#L34

Reply:
But that would require either a new (undocumented) dedicated command to open R with the library (`nonpareil-R`?), or a command that overrides the `R` in the environment, no?
I see how that works in circos, since circos itself is using the perl library, but in this case Nonpareil is not using the R library, the final user directly calls the functions of the library from R.

Reply:
> Nonpareil is not using the R library, the final user directly calls the functions of the library from R.

Ah, I see. In that case, I think what you have now is probably best.